src/routes/papers/index.js

```diff
@@ -41,28 +34,57 @@ function getBadgeCount(request, response) {
   });
 }
 
+
 function getBadgesByType(request, response) {
-  if (!request.params.doi1 || !request.params.doi2) {
+  if (!request.params.doi1 || !request.params.doi2 || !request.params.badge) {
     response.status(400).end();
     return;
   }
 
-  returnBadges(badgerService.getBadges(null, request.params.badge, {
+  return badgerService.getBadges(null, request.params.badge, {
     '_1': request.params.doi1,
     '_2': request.params.doi2
-  }), request, response);
+  });
 }
 
-function getUserBadges(request, response) {
+function getBadgesByBadge(request, response) {
```

Comment:
I like how the function names are consistent now! I don't like how the name is more confusing now.

I *think* I'm okay with this change, but I'll think a bit more.